forms-html/Hameed-Sara_NoorAldin/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hameed-Sara_NoorAldin</title>
+</head>
+<body>
+    
+    <form action="POST">
+        <br>
+        <label for="email">Enter your email:</label>
+        <input type="email" name="Email" placeholder="enter you email" required>
+        <br>
+        <br>
+        <label for="username">Username:</label>
+        <input type="text" name="Username"  placeholder="enter your name">
+        <br>
+        <br>
+        <label for="password">Username:</label>
+        <input type="password" name="password"  placeholder="enter your password">
+        <br>
+        <br>
+        <label for="quantity">Quantity (between 1 & 5):</label>
+        <input type="number" max="5" min="1" name="quantity" >
+        <br>
+        <br>
+        <label for="range">Volume (between 0 and 50):</label>
+        <input type="range" name="range" min="0" max="50" >
+        <br>
+        <br>
+        <br>
+        <!-- radio  -->
+        <h3>radio</h3>
+        <br>
+        <input type="radio" name="radio" id="radio1">
+        <label for="radio1">Male</label>
+        <br>
+        <input type="radio" name="radio" id="radio2">
+        <label for="radio2">Female</label>
+        <br>
+        <input type="radio" name="radio" id="radio3">
+        <label for="radio3">Other</label>
+        <br>
+        <br>
+        <!-- checkbox  -->
+        <h3>checkbox</h3>
+        <br>
+        <input type="checkbox" name="checkbox" id="checkbox1">
+        <label for="checkbox1">I have a bike</label>
+        <br>
+        <input type="checkbox" name="checkbox" id="checkbox2">
+        <label for="checkbox2">I have a car</label>
+        <br>
+        <input type="checkbox" name="checkbox" id="checkbox3">
+        <label for="checkbox3">I have a boat</label>
+        <br>
+        <br>
+        <!-- button  -->
+        <h3>button</h3>
+        <br>
+        <input type="button" value="Click Me!">
+        <br>
+        <label for="color">Selece your favorite color</label>
+        <input type="color" name="color" id="color">
+        <br>
+        <br>
+        <label for="date">Enter a date before 1980-01-01:</label>
+        <input type="date" name="date" id="date" min="1980-01-01">
```

Comment:
the label says **Enter a date before 1980-01-01** so the `max` value what we should use here and not `min`.